metrics.go

```diff
@@ -84,6 +84,16 @@ var (
 		Name:      "site_mppt_current_dc",
 		Help:      "Site mppt current DC in A",
 	}, []string{"inverter", "mppt"})
+	meterEnergyRealSumConsumedVec = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "meter_energy_consumed_wh",
+		Help:      "Meter consumed energy from grid in Wh",
+	}, []string{"device"})
+	meterEnergyRealSumProducedVec = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "meter_energy_produced_wh",
+		Help:      "Meter produced energy to grid in Wh",
```

Comment:
If the metrics are a Sum, then we probably should append the total suffix. WDYT? https://prometheus.io/docs/practices/naming/
```suggestion
	meterEnergyRealSumConsumedVec = promauto.NewGaugeVec(prometheus.GaugeOpts{
		Namespace: namespace,
		Name:      "meter_energy_consumed_wh_total",
		Help:      "Meter consumed energy from grid in Wh",
	}, []string{"device"})
	meterEnergyRealSumProducedVec = promauto.NewGaugeVec(prometheus.GaugeOpts{
		Namespace: namespace,
		Name:      "meter_energy_produced_wh_total",
		Help:      "Meter produced energy to grid in Wh",
```